templates/about.html

```diff
@@ -10,10 +10,20 @@
             <a href="/about">About</a>
         </nav>
     </header>
-    <div>
-      <h1>Background</h1>
-      <p><a href="https://github.com/w3c/web-platform-tests">web-platform-tests</a> is a test suite for the web platform.</p>
-    </div>
+    <article>
+      <h1>About</h1>
+      <p>The WPT Dashboard, <a href="http://wpt.fyi/">wpt.fyi</a>, displays results for the <a href="https://github.com/w3c/web-platform-tests">web-platform-tests</a>, or WPT, which are a group of test suites for many web platform specifications (each top-level directory of WPT is a different specification).</p>
+      <p>If the web platform as a whole were an engineering project, it'd be crazy not to have a test suite that's run regularly. The goal of the WPT Dashboard is to promote viewing the web platform as one entity and to make identifying and fixing interoperability issues as easy as possible.</p>
+      <p>The WPT Dashboard project is comprised of two parts:</p>
+      <ul>
+          <li><b>Running:</b> currently we try to run Chrome dev channel and Firefox nightly daily, with Edge and Safari soon to follow.</li>
+          <li><b>Serving:</b> this site, <a href="http://wpt.fyi/">wpt.fyi</a>, displaying JSON result files loaded from cloud storage.</li>
```

Comment:
- If you say "currently ..., with ... soon to follow", then this may go out of date soon, so if you use this wording we should be sure to update when it should be updated.
 - Mentioning that data is stored as JSON in clould storage seems like an implementation detail.